woocommerce/woocommerce-android
# [Product Selector] Add support for RTL languages in `WCSearchField`

## RTL languages support in `WCSearchField` in Product Selector

Closes: #8769 


### Description

The cursor was not working correctly with RTL languages.

### Testing instructions

1. Go to products selector (e.g. Orders > "+" > "+ Add new products")
2. Verify the cursor position is correct in case of RTL language

### Images/gif

| Before | After |
| ------------- | ------------- |
| <video src=https://user-images.githubusercontent.com/4527432/230627407-e35e9222-c5ee-4c22-95cb-574d8b0f6bd9.mp4/>  | <video src=https://user-images.githubusercontent.com/4527432/230627427-159e2f62-60bb-4c65-8cc8-8dd54f486db8.mp4/>  |
